src/eradiate/util/misc.py

```diff
@@ -465,8 +465,17 @@ def summary_repr(value):
     return repr(value)
 
 
+@summary_repr.register
+def _(datasets: list):
+    return [_summary_repr_impl_ds(ds) for ds in datasets]
```

Comment:
This returns a `list[str]`, not a repr (which should be a `str`). I'd be more in favour of using a lambda to do this where you need it.

Reply:
You are right; this function is obsolete actually